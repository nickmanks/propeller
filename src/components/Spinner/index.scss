@import '#scss/colors';

.spinner {
  animation: spin 2s linear infinite;
  border: 16px solid #f3f3f3;
  border-radius: 50%;
  border-top: 16px solid $primary;
  height: 150px;
  height: 120px;
  left: 50%;
  margin: -75px 0 0 -75px;
  position: absolute;
  top: 50%;
  width: 150px;
  width: 120px;
  z-index: 1;
}

@keyframes spin {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}
