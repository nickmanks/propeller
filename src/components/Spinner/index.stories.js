import React from 'react';
import Spinner from '.';
import {component} from '#src/testing/snapshots';

component('Spinner').when('standard', ()=> <Spinner />);
