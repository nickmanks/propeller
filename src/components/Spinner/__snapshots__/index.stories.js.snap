// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`standard 1`] = `
<div
  className="mock-spinner"
/>
`;
